packages/sanity/src/core/studio/components/navbar/StudioNavbar.tsx

```diff
@@ -160,7 +160,7 @@ export function StudioNavbar() {
       <RootCard borderBottom data-testid="navbar" data-ui="Navbar" padding={2} sizing="border">
         {/** Left flex */}
         <Flex align="center" justify="space-between">
-          <Flex align="center" gap={2}>
+          <Flex align="center" gap={2} style={{flexShrink: 0}}>
```

Comment:
Should we move this to a styled component for consistency?

Reply:
Yes! In general we try and avoid inline styles 

Though we'll likely want to move to a CSS grid approach for the navbar as we want the tool menu to be horizontally centered in the viewport regardless of the width of either the LHS or RHS sections. We can address this separately!